pkg/config/config.go

```diff
@@ -280,7 +280,9 @@ func InitConfig(config Config) {
 	config.BindEnvAndSetDefault("remote_configuration.enabled", false)
 	config.BindEnvAndSetDefault("remote_configuration.key", "")
 	config.BindEnv("remote_configuration.api_key")
-	config.BindEnv("remote_configuration.rc_dd_url", "")
+	config.BindEnv("remote_configuration.rc_dd_url")
+	config.BindEnvAndSetDefault("remote_configuration.rc_no_tls", false)
+	config.BindEnvAndSetDefault("remote_configuration.rc_no_tls_validation", false)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** Let's avoid double negation in the configuration names‏
```suggestion
	config.BindEnvAndSetDefault("remote_configuration.rc_require_tls", true)
	config.BindEnvAndSetDefault("remote_configuration.rc_require_tls_validation", true)
```

Reply:
followed the pattern of other products